### ISSUE TITLE
Fun Facts section is a bit misleading 

### ISSUE THREAD

**Issue by jwoertink** [sourcerer-io/sourcerer-app#466]
I think this data is pretty cool, but the "Fun Facts" section feels a little misleading to me.

Using [my page](https://sourcerer.io/jwoertink) as an example:

1. I definitely do *not* prefer camelcase. 
2. I've used ruby for closer to 15 years, but SVN was a thing before github existed 😛 
3. Same for JS, CSS, HTML
4. My most "starred" repo is in Crystal, as well as I contribute to mainly Crystal repos these days. What would be the metric for it say how long I've been using Crystal?
5. I think it's saying I'm most productive between noon and 6pm, but I take lunch from 12-1, and leave the office around 4:30pm... This may be accurate, but I'm not sold on it. I'm also not really sure why some of the bars in that graph are light gray and some are dark almost black.

On a side note, if I just create a javascript repo and accidentally committed my `node_modules` directory, would that really mess up this data? ... not that I've ever done that before 🤐

**Comment by sergey48k** [comment]
@yaronskaya - could you debug this case?

**Comment by yaronskaya** [comment]
Hi @jwoertink! Thank you for your comments.

1. We are debugging that.
2, 3. Currently, we don't support svn repositories, so the Ruby has 10 years.
4. We've added how long you've been using Crystal.
5. The dark means night time:) How does the graph should look like to your perception?

**Comment by revolter** [comment]
Maybe it should say something along the lines of "I've used Ruby for at least 10 years" and maybe add an info button that explains that the respective information is deduced from GitHub (or GitLab) activity and might not be 100% accurate.

**Comment by jwoertink** [comment]
Yeah, I like the "for at least", even though it does make it a bit verbose. Maybe could use ">= Xyrs"? I think it's all starting to make a bit more sense now. Thanks @yaronskaya!

**Comment by yaronskaya** [comment]
Fixed calculation of variable naming fact. Should be snake_case now.

We will add mention of possible older contributions soon. Thank you @jwoertink!

**Comment by jwoertink** [comment]
It's looking great! Thanks for the update.
